### PR TITLE
"Minor changes"

### PR DESCRIPTION
**Pull request by frivoal**
Attempts to address #243

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1921,10 +1921,18 @@ Working Group and Interest Group Charter Development</h4>
...
 
 	The Director <em class="rfc2119">must</em> solicit <a href="#ACReview">Advisory Committee review</a>
-	of every new or substantively modified Working Group or Interest Group charter.
-	The Director is <em class="rfc2119">not required</em> to solicit Advisory Committee review
-	prior to a charter extension or for minor changes.
+	of every new or substantively modified Working Group or Interest Group charter,
+	except for a charter extension,
+	or for substantive changes to a charter that do not affect the way the group functions in any significant way.
 	The review period <em class="rfc2119">must</em> be at least 28 days.</p>
+	The following are examples of substantive changes that would not require an AC Review:
```
Comment (nigelmegitt) [inline index.bs]: In the current Process doc the term "AC Review" is only used in the flow diagrams - suggest using "Advisory Committee Review" for consistency.

**Comment by dwsinger** [comment]
I think there is a horrible ambiguity here.

>The Director <em class="rfc2119">must</em> solicit <a href="#ACReview">Advisory Committee review</a> of every new or substantively modified Working Group or Interest Group charter, except for a charter extension, or for substantive changes to a charter that do not affect the way the group functions in any significant way.

Is the final "or" clause attached (a) to the first clause?, i.e.

>The Director <em class="rfc2119">must</em> solicit <a href="#ACReview">Advisory Committee review</a> of every new or substantively modified Working Group or Interest Group charter, or for substantive changes to a charter that do not affect the way the group functions in any significant way.


or to the 'except' clause 

>The Director <em class="rfc2119">must</em> solicit <a href="#ACReview">Advisory Committee review</a> of every new or substantively modified Working Group or Interest Group charter. Charter extensions, or substantive changes to a charter that do not affect the way the group functions in any significant way, do not require an AC review.

**Comment by chaals** [comment]
Dave Singer wrote:

> I think there is a horrible ambiguity here.

>> The Director must solicit Advisory Committee review of every new or  
>> substantively modified Working Group or Interest Group >>charter,  
>> except for a charter extension, or for substantive changes to a charter  
>> that do not affect the way the group functions in >>any significant way.
>
> Is the final "or" clause attached (a) to the first clause? [...] or to  
> the 'except' clause

I am certain it is to the second clause. If we rewrote the exceptions as a  
list:

> ...AC review for all charters except:
> * charter extensions
> * "minor" changes

**Comment by frivoal** [comment]
It's the second, otherwise it's nonsensical: substantively modified charter *or* substantive changes to a charter that [qualified that subsets the cases this applies to. That'd be equivalent not to have the or at all. I'm happy to turn it into bullet points as chaals suggested if you think it helps (I don't think it's needed, but I have no opposition to it.).

**Comment by dwsinger** [comment]
Except the first time I read it, I thought that the "not" had crept in by error.

**Comment by frivoal** [comment]
@dwsinger Clarified. Ready to merge now?

**Pull request merged**